pootle/core/utils/stats.py

```diff
@@ -26,3 +29,24 @@ def make_dict(state, title, filter_url=True):
     states.append(make_dict('untranslated', _("Untranslated")))
 
     return states
+
+
+def get_top_scorers_data(top_scorers, chunk_size):
+    all_scorers_loaded = len(top_scorers) <= chunk_size
+
+    top_scorers_data = [
+        dict(
```

Comment:
Yes @unho right (and this is the friendly reminder that spending too much consecutive hours working is not good 😫)